Libraries/PeriphDrivers/Source/PT/pt_reva.c

```diff
@@ -43,7 +43,7 @@ void MXC_PT_RevA_Init(mxc_ptg_reva_regs_t *ptg, mxc_clk_scale_t clk_scale)
     ptg->enable = 0;
 
     //clear all interrupts
-    ptg->intfl = ptg->intfl;
```

Comment:
Thanks. Just a note, even though parts without the ready_intfl/ready_inten registers will call the Init function, functionally, nothing should happen.

Reply:
Are you sure?  If `stop_intfl` is not a member of the register struct then won't the compiler throw an error trying to resolve it?  Also, do _all_ parts that use the Rev A PT files have split registers?

This could drive the compiler definition to wrap around the deprecated functions.  Something like `HAS_STOP_REG`?

```C
#ifdef HAS_STOP_REG
ptg->stop_intfl = ptf->stop_intfl;
ptf->ready_intfl = ptf->ready_intfl;
#else
ptg->intfl = ptg->intfl;
#endif
```

```C
#ifdef HAS_STOP_REG
 deprecated("Use MXC_PT_EnableStopInt or MXC_PT_EnableStartInt instead.  See pt.h for more details."))) void MXC_PT_EnableInt(uint32_t pts);
// ... other functions ...
#else
void MXC_PT_EnableInt(uint32_t pts);
#endif
```